testsuite/integration-arquillian/tests/other/console/pom.xml

```diff
@@ -32,7 +32,8 @@
     <name>Admin Console UI Tests</name>
 
     <properties>
-        <keycloak.theme.dir>${auth.server.home}/themes</keycloak.theme.dir>
+        <keycloak.theme.dir>${auth.server.home}/themes-primary</keycloak.theme.dir>
+        <developer.theme.dir>${project.basedir}/../../../../../themes/src/main/resources/theme</developer.theme.dir>
```

Comment:
Exactly what I'm saying. You're reading the themes from a different directory, so the setup doesn't match an actual real installation.